patches/gcc/0001-Add-Managarm-target.patch

```diff
@@ -14,69 +13,25 @@ TODO: Use libtoolize instead of modifying in-tree libtool?
  gcc/config/i386/i386-managarm.h             | 11 ++++
  gcc/config/i386/t-managarm64                |  4 ++
  gcc/config/managarm-kernel.h                |  9 +++
- gcc/config/managarm-system.h                |  3 +
+ gcc/config/managarm-mlibc.h                 |  3 +
  gcc/config/managarm.h                       | 13 ++++
  gcc/config/riscv/managarm.h                 | 64 +++++++++++++++++++
  gcc/config/riscv/t-managarm                 |  3 +
  libgcc/config.host                          | 20 ++++++
  libgcc/libgcov.h                            |  1 +
  libstdc++-v3/crossconfig.m4                 | 11 ++++
  libstdc++-v3/include/c_compatibility/fenv.h | 10 +--
- libtool.m4                                  | 14 ++++
- 18 files changed, 293 insertions(+), 8 deletions(-)
+ 16 files changed, 270 insertions(+), 6 deletions(-)
  create mode 100644 gcc/config/aarch64/aarch64-managarm.h
  create mode 100644 gcc/config/aarch64/t-aarch64-managarm
  create mode 100644 gcc/config/i386/i386-managarm.h
  create mode 100644 gcc/config/i386/t-managarm64
  create mode 100644 gcc/config/managarm-kernel.h
- create mode 100644 gcc/config/managarm-system.h
+ create mode 100644 gcc/config/managarm-mlibc.h
  create mode 100644 gcc/config/managarm.h
  create mode 100644 gcc/config/riscv/managarm.h
  create mode 100644 gcc/config/riscv/t-managarm
 
-diff --git a/config.sub b/config.sub
-index 63c1f1c8b..033763b58 100755
---- a/config.sub
-+++ b/config.sub
-@@ -132,7 +132,8 @@ case $1 in
- 		maybe_os=$field2-$field3
- 		case $maybe_os in
- 			nto-qnx* | linux-* | uclinux-uclibc* \
--			| uclinux-gnu* | kfreebsd*-gnu* | knetbsd*-gnu* | netbsd*-gnu* \
-+			| uclinux-gnu* | managarm-kernel* | managarm-system* \
```

Comment:
Does the upstream Managarm target use `-system` or `-mlibc`? Is the plan to switch to `-mlibc`?

Reply:
Yes, that is what landed in config.sub upstream.